scripts/update_docs.sh

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ev
+
+# Build docs
+function build_docs {
+    rm -rf docs/build/
+    make html
+    return $?
+}
+
+build_docs
+
+cp -R docs/build/html/* docs/
```

Comment:
Maybe delete `docs/build` after the copy?

Reply:
Deleted.